doc/dune-files.rst

```diff
@@ -482,98 +480,96 @@ in the future.
+  of and optionally followed by a dot, then anything else you want. The package
+  name must also be one of the packages that Dune knows about, as determined by
+  the :ref:`opam-files`
 
-- ``(package <package>)`` installs a private library under the specified package.
-  Such a library is now usable by public libraries defined in the same project.
-  The Findlib name for this library will be ``<package>.__private__.<name>``;
-  however, the library's interface will be hidden from consumers outside the
-  project.
+- ``(package <package>)`` installs a private library under the specified
+  package. Such a library is now usable by public libraries defined in the same
+  project. The Findlib name for this library will be
+  ``<package>.__private__.<name>``; however, the library's interface will be
+  hidden from consumers outside the project.
 
 - ``(synopsis <string>)`` should give a one-line description of the library.
   This is used by tools that list installed libraries
 
 - ``(modules <modules>)`` specifies what modules are part of the library. By
   default, Dune will use all the ``.ml/.re`` files in the same directory as the
-  ``dune`` file. This includes ones present in the file system as well
-  as ones generated by user rules. You can restrict this list by using a
-  ``(modules <modules>)`` field. ``<modules>`` uses the :ref:`ordered-set-language`,
-  where elements are module names and don't need to start with an uppercase
-  letter. For instance, to exclude module ``Foo``, use ``(modules (:standard \
-  foo))``
+  ``dune`` file. This includes ones present in the file system as well as ones
+  generated by user rules. You can restrict this list by using a ``(modules
+  <modules>)`` field. ``<modules>`` uses the :ref:`ordered-set-language`, where
+  elements are module names and don't need to start with an uppercase letter.
+  For instance, to exclude module ``Foo``, use ``(modules (:standard \ foo))``
 
 - ``(libraries <library-dependencies>)`` specifies the library's dependencies.
   See the section about :ref:`library-deps` for more details.
 
 - ``(wrapped <boolean>)`` specifies whether the library modules should be
-  available only through the top-level library module, or if they should all be exposed
-  at the top level. The default is ``true``, and it's highly recommended to keep
-  it this way. Because OCaml top-level modules must all be unique when linking
-  an executables, polluting the top-level namespace will make your library
-  unusable with other libraries if there is a module name clash. This option is
-  only intended for libraries that manually prefix all their modules by the
-  library name and to ease porting of existing projects to Dune.
+  available only through the top-level library module, or if they should all be
+  exposed at the top level. The default is ``true``, and it's highly recommended
+  to keep it this way. Because OCaml top-level modules must all be unique when
```

Comment:
OCaml Toplevel or generic toplevel, or the highest (top) level?
Same question throughout this section…